### PR TITLE
SAM-2693: Empty menu item when viewing submission for assessment with…

### PR DESCRIPTION
**Pull request by cschauer**
… random draw part

Avoid creating an empty li element that results in an empty menu item.

### CODE REVIEW

**Comment by cschauer** [comment]
This is really trivial, but we were asked to fix it locally because the empty menu item looked bad with our skin. You can't even tell that it's there on the default skin.

File: samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp [hunk 1]
```diff
@@ -134,7 +134,7 @@ function toPoint(id)
       <f:actionListener
         type="org.sakaiproject.tool.assessment.ui.listener.evaluation.QuestionScoreListener" />
     </h:commandLink>
-    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne ''}" /> 
+    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne '' && !totalScores.hasRandomDrawPart}" />
```
Comment (jonespm) [inline samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp]: I think you're going to have to split out the </span> here to just be on the #{totalScores.firstItem ne ''}

Also the closing </span><li> down below seems like should probably also be protected the same way so this is balanced?

Comment (cschauer) [inline samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp]: The last closing has to be there to close the last menu item whether it's this one or the one before it.

This whole section is kind of hard to read. Looking at this again, I realized none of the li tags are even closed except for the last one. And it's not just a problem on this page, but on all the evaluation pages with these menu items. I think I'll just clean up this section a bit and see if I can make it reusable for all the other pages that create these menu items.

**Comment by ncaidin** [comment]
icla for @cschauer is on file.

**Comment by ottenhoff** [comment]
I think we're still waiting for an update here from @cschauer ?

**Pull request closed without merging**